src/components/ButtonPanel.js

```diff
@@ -3,33 +3,19 @@ import Button from './Button';
 const ButtonPanel = () => (
   <div>
     <div className="btn-group">
-      <Button buttonName="AC" />
-      <Button buttonName="+/-" />
-      <Button buttonName="%" />
-      <Button buttonName="÷" />
+      {['AC', '+/-', '%', '÷'].map(el => (<Button key={el[0]} buttonName={el} />))}
```

Comment:
- [ ] **[OPTONAL]** Excellent job DRYing your code here. We can even use `object.keys` to loop through objects if you have a variable defined as follows:
```js
const buttonNames = { 'Group-1': ['AC', '+/-', '%', '÷'], 'Group-2': ['7', '8', '9', 'X'], ........}
```
use [objects keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) to loop through each objects (outer loop), and `map` to loop through each object array values (inner loop).

If you don't like to use objects you can define the variable as a [multidimensional array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#creating_a_two-dimensional_array) and use `map` for both outer and inner loops.